demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -317,6 +317,41 @@ def generate_readmes_for_new_content_pack(self, is_contribution=False):
                     if file_name.startswith('playbook') and file_name.endswith('.yml'):
                         self.generate_readme_for_pack_content_item(file)
 
+    def rearranging_before_conversion(self) -> None:
+        """
+        This method was written in order to fix a bug that exists in the server,
+        which incorrectly maps indicatorfield to incidentfield directory.
+        If this happened, this method will map the indicatorfield files to a separate folder called - indicatorfiled.
+        """
+        unpacked_contribution_dirs = get_child_directories(self.pack_dir_path)
+
+        for unpacked_contribution_dir in unpacked_contribution_dirs:
+
+            dir_name = os.path.basename(unpacked_contribution_dir)
+
+            # incidentfield directory may contain indicator-fields files
+            if dir_name == FileType.INCIDENT_FIELD.value:
+
+                dst_dir_path = None
+                is_first_file = True
```

Comment:
Why do you need this condition? `dst_dir_path` None.

Reply:
I need to initialize it beforehand, otherwise there is a comment

Reply:
I fixed it